index.html

```diff
@@ -5330,49 +5329,85 @@ <h2>Service Privacy</h2>
-mediator, or proxy that can provide herd privacy. For the third category,
-avoid the use of multiple <a>service endpoints</a> for a <a>DID</a> because some
-of these (e.g., an authorization server) are likely to be reused with other,
-related <a>DIDs</a>. Place correlatable <a>service endpoints</a> behind a
-privacy-preserving  mechanism, if possible, or introduce a mediator or proxy
-as a sole <a>service endpoint</a> in order to obscure related devices and documents
-through herd privacy.
+The degree of additional privacy risk caused by using multiple <a>service
+endpoints</a> in one <a>DID document</a> can be difficult to estimate. Privacy
+harms are typically unintended consequences. <a>DIDs</a> can identify documents,
+<a>services</a>, schemas, and other things that may be associated
+with individual people, households, clubs, and employers &mdash; and
+correlation of their <a>service endpoints</a> could become a powerful
+surveillance and inference tool.
       </p>
+<p>
+The variety of possible endpoints makes it particularly challenging to maintain 
+herd privacy, in which no information about the <a>DID subject</a> is leaked.
+</p>
+<p>
+First, because service endpoints may be specified as <a>URIs</a>, they could 
+unintentionally leak personal information because of the architecture of the service. 
+For example, a service endpoint of <code>http://example.com/MyFirstName</code> is leaking the term 
+<code>MyFirstName</code> to everyone who can access the DID Document. When linking to legacy systems, 
+this is an unavoidable risk and care should be taken in such cases. We encourage new, 
+DID-aware endpoints to use nothing more than the DID itself for any identification necessary,
+e.g., <code>http://example.com/did%3Aexample%3Aabc123</code>. Since that <code>did:example:abc123</code> is already 
+exposed in the DID Document, it leaks no additional information.
+</p>
+
+<p>
+Second, because a <a>DID document</a> can list multiple service endpoints, it is possible to 
+irreversibly associate services that are not associated in any other context. This 
+correlation on its own may lead to de-anonymization, revealing information about the 
+<a>DID subject</a> even if the <a>URIs</a> used did not.
+</p>
+<p>
+Third, because some types of <a>DID subjects</a> may be more or less likely to list specific 
+endpoints, e.g., a <a>DID</a> for an automobile may include a pointer to a public title 
+record at the Department of Motor Vehicles, while a <a>DID</a> for an individual would not.
+As such, the listing of a given service could, by itself, leak information that can 
+be used to infer something about the subject.
+</p>
+<p>
+It is the goal of herd privacy to ensure that the nature of specific subjects is 
+obscured by the population of the whole. To maximize herd privacy, implementers 
+need to rely on one &mdash; and only one &mdash; service endpoint, with that endpoint providing 
+a proxy or mediator service to further blind requests.
```

Comment:
```suggestion
a proxy or mediator service who is trusted enough to be informed about these correlating attributes to further blind requests.
```

Reply:
usage of the service endpoint doesn't completely eliminate the correlation concern, but it significantly reduces who can make the inference. This addition is trying to make that clear to readers/implementors so there's no false sense of privacy being portrayed.

Reply:
Let me know if the fix works for you.